gson/src/main/java/com/google/gson/Gson.java

```diff
@@ -410,6 +410,10 @@ private void checkValidFloatingPoint(double value) {
    *  System.out.println("Num JSON reads" + stats.numReads);
    *  System.out.println("Num JSON writes" + stats.numWrites);
    *  }</pre>
+   *  Note that this call will skip all factories registered before @param skipPast. In case of
```

Comment:
Nope it doesn't, my bad I was tricked by vim highlighting it. I guess `@code` will have to do.